linera-execution/src/wasm/conversions_to_wit.rs

```diff
@@ -158,52 +158,48 @@ impl From<ChainId> for service::ChainId {
 
 impl From<CryptoHash> for contract::CryptoHash {
     fn from(crypto_hash: CryptoHash) -> Self {
-        let bytes = crypto_hash.as_bytes();
-
+        let [part1, part2, part3, part4]: [u64; 4] = crypto_hash.into();
```

Comment:
curious if the type is really needed

Reply:
It is! I'm also surprised that it can't be inferred.